subscription/src/subscription_worker.py

```
@@ -39,7 +38,7 @@ def delete_messages(sqs_client, queue_url, messages):
         receipt_handle = message['ReceiptHandle']
         delete_message(sqs_client=sqs_client, queue_url=queue_url, receipt_handle=receipt_handle)
 
-def process_messages(sns_client, topic, messages, access_control, search):
+def process_messages(sns_client, topic, messages, access_control):
```

Ruff (0.13.1)

Unused function argument: `access_control`

(ARG001)